### PR TITLE
Typescript and CI

### LINKED ISSUE #3: Continuous support
I made some improvements
- Typescript support
- Renovate Bot for updating dependencies
- Travis CI build with weekly cron
- fixed missing strings
- removed not working image from README
- other minor changes

would you be interested in merging it?
BTW, is this translation still sponsored and supported, as README states?

### PR DESCRIPTION
**Pull request by Tymek**
closes #3 
published as `@scrlk/ra-language-polish`

### CODE REVIEW

**Comment by Tymek** [comment]
@tskorupka seems like I do have access to merge it myself, without review :confused: 

anyway, I don't think I have permission to publish it in npm. this part can be automated

File: src/index.ts [hunk 1]
```diff
@@ -75,6 +80,7 @@ module.exports = {
             no: 'Nie',
             not_found: 'Wpisałeś zły URL lub zostałeś niepoprawnie przeniesiony.',
             yes: 'Tak',
+            unsaved_changes: 'Niektóre zmiany nie zostały zapisane. Czy jesteś pewiem, że chcesz je zignorować?',
```
Comment (tskorupka) [inline src/index.ts]:
```suggestion
            unsaved_changes: 'Niektóre zmiany nie zostały zapisane. Czy jesteś pewien, że chcesz je zignorować?',
```

**Review by tskorupka — CHANGES_REQUESTED** [review]
One typo fix is needed

**Review by Tymek — COMMENTED** [review]
@tskorupka typo fixed. I also updated misleading license statement in README

**Review by tskorupka — APPROVED** [review]

**Comment by Tymek** [comment]
@tskorupka I don't see new version published on npm. Could you add me as a package maintainer? [npmjs.com/~tymek](https://www.npmjs.com/~tymek)

**Pull request merged**